src/AudioDevice.cpp

```diff
@@ -36,6 +36,7 @@
 
 // Log tag for AudioDevice.
 #define TAG "AUDIO"
+#define DEFAULT_PCM "plughw:0,0"
```

Comment:
Hard coding platform specific card name is not recommended. Please see [SJ's feedback](https://github.com/intel/earlyapp/commit/d885d2d209cb3b2bb73fd992680a0cd2a3c0d652#commitcomment-31970860).

Reply:
Back to "default" to avoid impact other plt, seek ALSA configuration to resolve it.